app/assets/stylesheets/components/page_specific/_home_top.scss

```diff
@@ -0,0 +1,193 @@
+// Banner at the top of the home page
+//
+// Styleguide Home Top
+
+.home-top {
+  background: image_url('illustrated_people.png') 0 100% no-repeat;
+  background-size: 135%;
```

Comment:
There was a lot of adjustments made to ensure that the people's heads don't interfere with the trust text, and it looks like the design as much as possible!